src/referencePackages/src/system.componentmodel.composition/4.5.0/System.ComponentModel.Composition.4.5.0.csproj

```diff
@@ -0,0 +1,32 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))/dir.props" />
+
+  <PropertyGroup>
+    <TargetFrameworks>netstandard2.0</TargetFrameworks>
+    <NuspecFile>$(ArtifactsBinDir)system.componentmodel.composition/4.5.0/system.componentmodel.composition.nuspec</NuspecFile>
+    <StrongNameKeyId>ECMA</StrongNameKeyId>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <OutputPath>$(ArtifactsBinDir)system.componentmodel.composition/4.5.0/ref/</OutputPath>
+    <IntermediateOutputPath>$(ArtifactsObjDir)system.componentmodel.composition/4.5.0</IntermediateOutputPath>
+  </PropertyGroup>
+
+  <PropertyGroup Condition=" '$(TargetFramework)' == 'netcoreapp2.0' ">
+    <OutputPath>$(ArtifactsBinDir)system.componentmodel.composition/4.5.0/lib/</OutputPath>
```

Comment:
Is this needed?  IIRC you needed to remove the netcoreapp2.0 tfm.  Same goes for the System.Security.Permissons reference.  That might remove the need for that package completely. and possibly others.  Also the lib/netcoreapp2.0 directory sould be cleaned up.